### PR TITLE
add drop down for states

### PR DESCRIPTION
**Pull request by amazingshellyyy**

### CODE REVIEW

File: client/src/components/covid19/CACounty.js [hunk 1]
```diff
@@ -27,21 +60,40 @@ class CACounty extends React.Component {
...
+            </div>
+            <div className="col-sm-4">
+              <Dropdown className="ml-auto">
+                <Dropdown.Toggle variant="outline-secondary" id="dropdown-basic">
+                  {this.state.curState}
+                </Dropdown.Toggle>
+                <Dropdown.Menu onClick={this.choose.bind(this)}>
+                  <Dropdown.Item id="CA" href="">California</Dropdown.Item>
+                  <Dropdown.Item id="WA" href="">Washington</Dropdown.Item>
+                  <Dropdown.Item id="NY" href="">New York</Dropdown.Item>
```
Comment (amazingandyyy) [inline client/src/components/covid19/CACounty.js:81]: what is the `href=""` here

File: client/src/components/covid19/USData.js [hunk 2]
```diff
@@ -39,10 +39,10 @@ class USData extends React.Component {
                 <YAxis />
                 <Tooltip />
                 <Legend />
-                <Bar dataKey='confirmed' barSize={20} fill='#8884d8' />
-                <Line type='monotone' dataKey='confirmed' stroke='#6562a1' />
-                <Bar dataKey='deaths' barSize={20} fill='#82ca9d' />
-                <Line type='monotone' dataKey='deaths' stroke='#538265' />
+                <Bar dataKey='confirmed' barSize={20} fill='#75D6B1' />
+                <Line type='monotone' dataKey='confirmed' stroke='#75D6B1' />
+                <Bar dataKey='deaths' barSize={20} fill='#757272' />
```
Comment (amazingandyyy) [inline client/src/components/covid19/USData.js:44]: can you declare this color as a global variable
e.g.
```js
const primary_color = "#75D6B1 ";
const death_color = "#757272 ";
```

File: client/src/components/covid19/CACounty.js (hunk 1 above)
Comment (amazingshellyyy) [reply client/src/components/covid19/CACounty.js]: will fix

File: client/src/components/covid19/USData.js (hunk 2 above)
Comment (amazingshellyyy) [reply client/src/components/covid19/USData.js:44]: color is not fix yet. will fix them and put in global variable  in the next pr

**Pull request merged**